puppet/modules/quickstack/manifests/cinder_volume.pp

```diff
@@ -22,6 +24,16 @@
   $nfs_shares             = [],
   $nfs_mount_options      = undef,
 
+  $dell_sc_san_ip           = [''],
+  $dell_sc_san_logi         = ['Admin'],
```

Comment:
Fixed it. There was some merge conflicts and I might have introduced this error.